it/uk/gov/hmrc/excisemovementcontrolsystemapi/DraftExciseMovementControllerItSpec.scala

```diff
@@ -20,9 +20,10 @@ import com.github.tomakehurst.wiremock.client.WireMock
 import com.github.tomakehurst.wiremock.client.WireMock._
 import org.bson.types.ObjectId
 import org.mockito.ArgumentMatchersSugar.any
-import org.mockito.MockitoSugar.when
+import org.mockito.MockitoSugar.{when, reset => MockitSugerReset}
```

Comment:
typo in MockitoSugarReset? Does it need an alias?

Reply:
Yes it need an Alias because it is getting confuse with wireMock.reset